From bb31ab1cfcbe93679f5497ac99824e6c1ec853b8 Mon Sep 17 00:00:00 2001
From: Michael McAuliffe <michael.e.mcauliffe@gmail.com>
Date: Mon, 18 Oct 2021 18:25:44 -0700
Subject: [PATCH] Add flag for version so it's not computed from git history

---
 CMakeLists.txt | 13 ++++++++-----
 1 file changed, 8 insertions(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8b2de3a0b..5d28cc6ad 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -55,6 +55,7 @@ endif()
 option(KALDI_BUILD_EXE "If disabled, will make add_kaldi_executable a no-op" ON)
 option(KALDI_BUILD_TEST "If disabled, will make add_kaldi_test_executable a no-op" ON)
 option(KALDI_USE_PATCH_NUMBER "Use MAJOR.MINOR.PATCH format, otherwise MAJOR.MINOR" OFF)
+set(OVERRIDE_KALDI_VERSION "" CACHE STRING "Use externally supplied Kaldi version")

 if (KALDI_BUILD_TEST)
     include(CTest)
@@ -165,11 +166,13 @@ if(CUDA_FOUND)
 endif()

 add_definitions(-DKALDI_NO_PORTAUDIO=1)
-
-include(VersionHelper)
-get_version() # this will set KALDI_VERSION and KALDI_PATCH_NUMBER
-if(${KALDI_USE_PATCH_NUMBER})
-    set(KALDI_VERSION "${KALDI_VERSION}.${KALDI_PATCH_NUMBER}")
+if(${OVERRIDE_KALDI_VERSION})
+    set(KALDI_VERSION "${OVERRIDE_KALDI_VERSION}")
+else()
+    include(VersionHelper)
+    get_version() # this will set KALDI_VERSION and KALDI_PATCH_NUMBER
+    if(${KALDI_USE_PATCH_NUMBER})
+        set(KALDI_VERSION "${KALDI_VERSION}.${KALDI_PATCH_NUMBER}")
 endif()

 # get_third_party(openfst)
--
2.32.0

