From 1421700872afb422263cff330634088fc602e6b3 Mon Sep 17 00:00:00 2001
From: Michael McAuliffe <michael.e.mcauliffe@gmail.com>
Date: Sun, 17 Apr 2022 08:47:02 -0700
Subject: [PATCH] Update cmake

---
 cmake/gen_cmake_skeleton.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/gen_cmake_skeleton.py b/cmake/gen_cmake_skeleton.py
index 8056786..fa9c515 100644
--- a/cmake/gen_cmake_skeleton.py
+++ b/cmake/gen_cmake_skeleton.py
@@ -92,7 +92,7 @@ def get_exe_additional_depends(t):
         "gmm-decode-*": ["decoder"],
         "gmm-align": ["decoder"],
         "gmm-align-compiled": ["decoder"],
-        "gmm-est-fmllr-gpost": ["sgmm2", "hmm"],
+        "gmm-est-fmllr-gpost": ["hmm"],
         "gmm-rescore-lattice": ["hmm", "lat"],
 
         # solve fgmmbin
-- 
2.35.2

