From dc93b01ef725516529cc128ad6bd772e6fde6f73 Mon Sep 17 00:00:00 2001
From: Michael McAuliffe <michael.e.mcauliffe@gmail.com>
Date: Tue, 16 Aug 2022 15:48:03 -0700
Subject: [PATCH] Remove Windows executables

---
 cmake/third_party/openfst.cmake | 56 ---------------------------------
 1 file changed, 56 deletions(-)

diff --git a/cmake/third_party/openfst.cmake b/cmake/third_party/openfst.cmake
index f9fff8a..d6ab794 100644
--- a/cmake/third_party/openfst.cmake
+++ b/cmake/third_party/openfst.cmake
@@ -17,62 +17,6 @@ if(NOT openfst_POPULATED)
 
     if(CONDA_ROOT)
 
-        if(MSVC)
-            install(TARGETS fstarcsort fstclosure fstcompile fstcompose fstconcat fstconnect fstconvert fstdeterminize
-                            fstdifference fstdisambiguate fstdraw fstencode fstepsnormalize fstequal fstequivalent
-                            fstinfo fstintersect fstinvert fstisomorphic fstmap fstminimize fstprint fstproject fstprune
-                            fstpush fstrandgen fstrelabel fstreplace fstreverse fstreweight fstrmepsilon fstshortestdistance
-                            fstshortestpath fstsymbols fstsynchronize fsttopsort fstunion
-            RUNTIME
-                DESTINATION ${CMAKE_INSTALL_BINDIR}
-                COMPONENT kaldi
-            )
-            install(TARGETS farcompilestrings farcreate farequal farextract farinfo farisomorphic farprintstrings
-            RUNTIME
-                DESTINATION ${CMAKE_INSTALL_BINDIR}
-                COMPONENT kaldi
-            )
-            install(TARGETS fstlinear fstloglinearapply
-            RUNTIME
-                DESTINATION ${CMAKE_INSTALL_BINDIR}
-                COMPONENT kaldi
-            )
-            install(TARGETS mpdtcompose mpdtexpand mpdtinfo mpdtreverse
-            RUNTIME
-                DESTINATION ${CMAKE_INSTALL_BINDIR}
-                COMPONENT kaldi
-            )
-            install(TARGETS pdtcompose pdtexpand pdtinfo pdtreplace pdtreverse pdtshortestpath
-            RUNTIME
-                DESTINATION ${CMAKE_INSTALL_BINDIR}
-                COMPONENT kaldi
-            )
-            install(TARGETS fstspecial
-            RUNTIME
-                DESTINATION ${CMAKE_INSTALL_BINDIR}
-                COMPONENT kaldi
-            )
-
-            install(DIRECTORY ${openfst_SOURCE_DIR}/src/include/ DESTINATION include/
-                    COMPONENT kaldi
-                    FILES_MATCHING PATTERN "*.h")
-
-            install(TARGETS fst
-                    EXPORT kaldi-targets
-                    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT kaldi
-                    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT kaldi NAMELINK_SKIP
-                    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT kaldi)
-        else()
-
-            install(TARGETS fst
-                    LIBRARY
-                        DESTINATION ${CMAKE_INSTALL_LIBDIR}
-                        COMPONENT kaldi
-                        NAMELINK_SKIP # Link to so with version to avoid conflicts with OpenFst 1.8.1 on conda
-                    )
-
-        endif()
-
     else() # Original functionality
 
         install(DIRECTORY ${openfst_SOURCE_DIR}/src/include/ DESTINATION include/
-- 
2.37.1

