From 81c9c3d2fcca3e82042b5160cbc65134fbaf5181 Mon Sep 17 00:00:00 2001
From: Michael McAuliffe <michael.e.mcauliffe@gmail.com>
Date: Tue, 16 Aug 2022 16:29:25 -0700
Subject: [PATCH] Fix CMake skeleton bug

---
 cmake/gen_cmake_skeleton.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/gen_cmake_skeleton.py b/cmake/gen_cmake_skeleton.py
index 81c4be8..5925c63 100644
--- a/cmake/gen_cmake_skeleton.py
+++ b/cmake/gen_cmake_skeleton.py
@@ -147,7 +147,7 @@ def get_exe_additional_depends(t):
     for pattern in additional.keys():
         if fnmatch.fnmatch(t, pattern):
             libs.extend(list(map(lambda name: lib_dir_name_to_lib_target(name), additional[pattern])))
-    return sorted(list(set(l)))
+    return sorted(list(set(libs)))
 
 def disable_for_win32(t):
     disabled = [
-- 
2.37.1

