From 916f4b3289607510896a7083b7241892912c2e1c Mon Sep 17 00:00:00 2001
From: Michael McAuliffe <michael.e.mcauliffe@gmail.com>
Date: Thu, 1 Feb 2024 10:41:44 -0800
Subject: [PATCH] Cuda 12 support

---
 CMakeLists.txt | 14 ++++++++++----
 1 file changed, 10 insertions(+), 4 deletions(-)

diff --git CMakeLists.txt CMakeLists.txt
index 24cb574a6..48a28fbfe 100644
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.18)
+cmake_minimum_required(VERSION 3.26)
 project(kaldi)
 
 if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
@@ -16,6 +16,12 @@ if(CONDA_ROOT)
     include_directories("${CONDA_ROOT}/include")
     if (NOT CUDA_TOOLKIT_ROOT_DIR)
         set(CUDA_TOOLKIT_ROOT_DIR "${CONDA_ROOT}")
+    else()
+        set(CMAKE_INCLUDE_PATH "${CMAKE_INCLUDE_PATH};${CUDA_TOOLKIT_ROOT_DIR}/include")
+        set(CMAKE_LIBRARY_PATH "${CMAKE_LIBRARY_PATH};${CUDA_TOOLKIT_ROOT_DIR}/lib;${CUDA_TOOLKIT_ROOT_DIR}/lib/stubs")
+        link_directories("${CUDA_TOOLKIT_ROOT_DIR}/lib")
+        link_directories("${CUDA_TOOLKIT_ROOT_DIR}/lib/stubs")
+        include_directories("${CUDA_TOOLKIT_ROOT_DIR}/include")
     endif()
 
 endif()
@@ -23,8 +29,8 @@ endif()
 include(third_party/get_third_party)
 
 
-find_package(PythonInterp)
-if(NOT PYTHON_EXECUTABLE)
+find_package(Python3 REQUIRED)
+if(NOT Python3_EXECUTABLE)
     message(FATAL_ERROR "Needs python to auto-generate most CMake files, but not found.")
 endif()
 
@@ -34,7 +40,7 @@ set(IS_LIB_SHARE "")
 if(BUILD_SHARED_LIBS)
     set(IS_LIB_SHARE "--shared")
 endif()
-execute_process(COMMAND ${PYTHON_EXECUTABLE}
+execute_process(COMMAND ${Python3_EXECUTABLE}
     "${CMAKE_CURRENT_SOURCE_DIR}/cmake/gen_cmake_skeleton.py"
     "${CMAKE_CURRENT_SOURCE_DIR}/src"
     "--quiet"
-- 
2.33.0.windows.1

