From 3ec56682ec70a4fc54309ae3a97ae6af77dd911b Mon Sep 17 00:00:00 2001
From: danijel3 <danijel@pja.edu.pl>
Date: Sun, 2 Jun 2024 23:11:33 +0200
Subject: [PATCH 7/7] Fix missing FLT_MAX in some CUDA installation scenarios.

---
 src/cudadecoder/cuda-decoder-kernels.cu | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/cudadecoder/cuda-decoder-kernels.cu b/src/cudadecoder/cuda-decoder-kernels.cu
index 8503182c1..e20a7dea1 100644
--- a/src/cudadecoder/cuda-decoder-kernels.cu
+++ b/src/cudadecoder/cuda-decoder-kernels.cu
@@ -26,6 +26,10 @@
 #include "cuda-decoder-kernels.h"
 #include "cuda-decoder-kernels-utils.h"
 
+#ifndef FLT_MAX
+#define FLT_MAX 340282346638528859811704183484516925440.0f
+#endif
+
 namespace kaldi {
 namespace cuda_decoder {
 
